ingest.go

```diff
@@ -1409,6 +1409,9 @@ func (d *DB) excise(
 			SmallestSeqNum: m.SmallestSeqNum,
 			LargestSeqNum:  m.LargestSeqNum,
 		}
+		leftFile.ValidateVirtual(m)
```

Comment:
This will be down where we add leftFile to `ve.NewFiles`, as most of its arguments are not set yet.